IdentityCore/src/telemetry/request_telemetry/MSIDLastRequestTelemetry.m

```diff
@@ -111,16 +160,29 @@ - (NSString *)telemetryString
     return result;
 }
 
-- (NSArray<MSIDRequestTelemetryErrorInfo *> *)errorsInfo
+#pragma mark - NSCoding
+
+#define kSchemaVersion              @"schemaVersion"
+#define kSilentSuccessfulCount      @"silentSuccessfulCount"
+#define kErrorsInfo                 @"errorsInfo"
+
+- (void)encodeWithCoder:(NSCoder *)encoder
 {
-    __block NSArray *errorsInfoCopy;
-    dispatch_sync(self.synchronizationQueue, ^{
-        errorsInfoCopy = [NSArray arrayWithArray:_errorsInfo];
-    });
-    return errorsInfoCopy;
+    [encoder encodeFloat:_schemaVersion forKey:kSchemaVersion];
+    [encoder encodeFloat:_silentSuccessfulCount forKey:kSilentSuccessfulCount];
```

Comment:
this shouldn't be float

Reply:
Thanks for pointing it out, I forgot to double check what types are specifically supported for encoding. Addressed in the next update.